### PR TITLE
Multiindex

### PR DESCRIPTION
**Pull request by stafyniaksacha**

### CODE REVIEW

**Comment by xbill82** [comment]
:+1:

**Comment by j33f** [comment]
I have a problem with manage indexes.
I have several indexes, only one of them have a collection.
When I choose any of the indexes from the selector in the menu, everything is ok
When I choose an index wich does not have a collection from the manage indexes page, the page show me the index wich have a collection...

**Comment by j33f** [comment]
after a proper bower install, npm install and grunt, everything went ok

**Comment by j33f** [comment]
+1

File: features/step_definitions/realtime/realtime.js [hunk 1]
```diff
@@ -16,33 +16,35 @@ var
...
       .call(callback);
   });
 
   this.Then(/^I click on the collection selector$/, function (callback) {
     browser
-      .pause(500)
-      .waitForVisible('drop-down-search .dropdown-toggle', 1000)
-      .click('drop-down-search .dropdown-toggle')
+      .waitForVisible('collections-drop-down-search .dropdown-toggle', 1000)
+      .pause(5000)
```
Comment (xbill82) [inline features/step_definitions/realtime/realtime.js]: We need less than that :smile:

File: features/step_definitions/realtime/realtime.js [hunk 2]
```diff
@@ -16,33 +16,35 @@ var
...
+      .click('collections-drop-down-search .dropdown-toggle')
       .call(callback)
   });
 
   this.Given(/^I click on a collection$/, function (callback) {
     browser
-      .waitForVisible('drop-down-search .dropdown-toggle', 1000)
-      .click('drop-down-search ul li:last-child a')
+      .waitForVisible('collections-drop-down-search .dropdown-menu', 4000)
```
Comment (xbill82) [inline features/step_definitions/realtime/realtime.js]: You can lower this down to 1000

**Pull request merged**